### PR TITLE
Travis ci

### LINKED ISSUE #3: Set up CI
- Use Travis
- Remove `.drone` files
- Publish to Docker Hub

### PR DESCRIPTION
**Pull request by balser**
Adds Travis config and removes Drone config. I'll configure the settings on Travis once the repo is made public.

Closes #3

### CODE REVIEW

**Comment by marzagao** [comment]
LGTM!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+go:
+  - 1.7
+
+env:
+  - CGO_ENABLED=0
+
+script:
+  - go vet
+  - go test
+  - go build -ldflags "-s -w -X main.build=$TRAVIS_BUILD_NUMBER -X main.rev=$TRAVIS_COMMIT" -a -tags netgo
```
Comment (tonglil) [inline .travis.yml]: I think `-X main.build=$TRAVIS_BUILD_NUMBER` is unnecessary, no `build` variable in main package.

That command is for building static binaries in earlier versions of go.

Try: `go build -a -ldflags "-X main.rev=$REV"`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.7
```
Comment (tonglil) [inline .travis.yml]: This should be `1.x` for latest stable version.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+env:
+  - CGO_ENABLED=0
```
Comment (tonglil) [inline .travis.yml]: Make sure to also set:
```
env:
  global:
    - CGO_ENABLED=0
    - GOOS=linux
    - GOARCH=amd64
```

File: .travis.yml (hunk 2 above)
Comment (fsouza) [inline .travis.yml]: Wait, does gimme support that? I tested locally and it didn't work x)

Comment (fsouza) [inline .travis.yml]: Oh, and one important thing with gimme: if you specify only 1.7, it's going to install 1.7, not the latest 1.7 release, so you probably wanna use ``1.7.4`` to make sure you have latest security fixes.

File: .travis.yml (hunk 3 above)
Comment (balser) [inline .travis.yml]: Good catch. Set in 5f0f2eb31cac3c5bc58d5961f6356d0856ecf9b7.

File: .travis.yml (hunk 1 above)
Comment (balser) [inline .travis.yml]: Ah, I think this drifted. Updated in 868059a89c9216d0e35854399b39c13c34c3096c. Thanks.

File: .travis.yml (hunk 2 above)
Comment (balser) [inline .travis.yml]: I went with `1.7.x` for now (68d5bc4f04bc55e9241431c618fcef6637dc7918) to avoid upgrading implicitly on unrelated changes.

**Comment by balser** [comment]
@tonglil @fsouza @marzagao thanks for the reviews, updated per comments.

**Comment by fsouza** [comment]
lgtm
